lab-said/src/main.js

```diff
@@ -0,0 +1,54 @@
+import './styles/main.scss';
+
+import React from 'react';
+import ReactDom from 'react-dom';
+import {BrowserRouter, Route} from 'react-router-dom';
+import AboutContainer from './component/about-container';
+import DashboardContainer from './component/dashboard-container';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      notes: [],
+    };
+    this.getApp = this.getApp.bind(this);
+  }
+
+  componentDidUpdate() {
+    console.log('__STATE__', this.state);
+    console.log('__NOTES__', this.state.noteList);
```

Comment:
this (line 20) is console logging as undefined. I don't think noteList exists at this point (in App)